about-us.html

```diff
@@ -0,0 +1,543 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Binabox</title>
+        <link rel="stylesheet" type="text/css" href="./css/styles.css">
+        <link rel="stylesheet" href="./css/modern-normalize.css">
```

Comment:
Please connect normalize before your own styles. It's very important, because first you are change browser styles with normalize and after that you can write your own styles.